content/departments/engineering/dev/enablement/frontend-platform/processes.md

```diff
@@ -53,6 +53,16 @@ If you think Frontend Platform is the right team to address a bug or other reque
 
 We use OpsGenie to manage a support rotation that changes weekly. The person on-call will be pinged when someone tags `@frontend-platform-support` in Slack. That person is expected to prioritize responding to the support request, but anyone who sees a request can and should jump in.
 
+#### Handling CI assistance requests
+
+If the `@frontend-platform-support`’s assistance was requested in one of the CI issues, this request should be handled in one of the following ways:
```

Comment:
I think you also want to offer some guidance on how to disable an flaky test, this way you're teaching others how to do this, lowering the amount of slack requests you're getting while being able to tackle the flakes on your schedule.

Reply:
[This PR](https://github.com/sourcegraph/sourcegraph/pull/37911) updates the flakes documentation, and it would be nice to link it directly to the Slack message. It has the information on [disabling flaky tests](https://docs.sourcegraph.com/dev/background-information/ci#flaky-tests). What do you think?

Reply:
I'll update our Slack auto-response 😄 Suggested rephrase for our process page:

If `frontend-platform-support` is pinged for a CI issue, the teammate on rotation should ask the person requesting help to [determine](https://github.com/sourcegraph/sourcegraph/pull/37911) whether or not the issue is due to a flaky test and to [disable it](https://docs.sourcegraph.com/dev/background-information/ci#flaky-tests) if so. 

Otherwise, the teammate on rotation should resolve the problem with a PR from the Frontend Platform team or create an issue and assign it to the appropriate owner. 

This process aims to _always_ have a solution or artifact for a reported problem so that we can avoid the same CI issues in the future.

Reply:
Thanks, @taylorsperry! I applied your suggestion 🙌